### PR TITLE
make cuttlefish work with erlang 20 / elixir 1.5.2

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: src/cuttlefish_escript.erl [hunk 1]
```diff
@@ -81,7 +81,7 @@ main(Args) ->
     application:set_env(lager, handlers, [{lager_stderr_backend, LogLevel}]),
     application:set_env(lager, crash_log, false),
     lager:start(),
-    lager:debug("Cuttlefish set to debug level logging"),
+    lager:log(debug,self(),"Cuttlefish set to debug level logging"),
```
Comment (ghost) [inline src/cuttlefish_escript.erl:84]: parse transform'd Erlang code won't work in elixir from 1.6 onwards - as there is minimal use of the lager parse transform it seemed reasonable to switch to direct lager calls

**Review by Licenser — APPROVED** [review]
👍 no reason not to merge

**Comment by Licenser** [comment]
@tsloughter any objections on your end?

**Comment by tsloughter** [comment]
Nope

On Mon, May 21, 2018, at 12:11 PM, Heinz N. Gies wrote:
> @tsloughter[1] any objections on your end?


> — You are receiving this because you were mentioned. Reply to this
> email directly, view it on GitHub[2], or mute the thread[3].

Links:

  1. https://github.com/tsloughter
  2. https://github.com/Kyorai/cuttlefish/pull/5#issuecomment-390736534

**Comment by Licenser** [comment]
published this as `2.1.0` to make sure people can opt out by using a `2.0.*`

**Pull request merged**